frontend/src/employees/TableListOfEmployees.js

```diff
@@ -24,6 +25,14 @@ export default function TableListOfEmployees() {
                   <td>{user.name}</td>
                   <td>{user.email}</td>
                   <td>{user.active}</td>
+                  <td>
+                    <Link to={`/employees/update/${user.id}`}>
+                      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-pencil-square" viewBox="0 0 16 16">
```

Comment:
Ezt az svg-t kiraknám minimum egy react komponensbe, hogy az elnevezéséből egyértelmű legyen, hogy mit is formáz ez a grafika. Egy másik lehetőség, hogy kirakod egy src/assets mappába egy .svg fájlként és onnan húzod be.